src/current/v23.1/changefeed-messages.md

```diff
@@ -248,6 +248,41 @@ There are three different ways to configure resolved timestamp messages:
 If you require `resolved` message frequency under `30s`, then you **must** set the [`min_checkpoint_frequency`](create-changefeed.html#min-checkpoint-frequency) option to at least the desired `resolved` frequency. This is because `resolved` messages will not be emitted more frequently than `min_checkpoint_frequency`, but may be emitted less frequently.
 {{site.data.alerts.end}}
 
+## Duplicate messages
+
+Under some circumstances, changefeeds will emit duplicate messages to ensure the sink is receiving each message at least once. The following can cause duplicate messages:
+
+- The changefeed job [encounters an error](#changefeed-encounters-an-error).
```

Comment:
Or is paused manually, as I think you cover elsewhere.

Reply:
I have added again here.